pkg/internal/checkup/executor/executor.go

```diff
@@ -96,7 +96,15 @@ func (e Executor) Execute(ctx context.Context, vmiName, podName, podContainerNam
 		return status.Results{}, fmt.Errorf("failed to clear trex stats on pod \"%s/%s\" side: %w", e.namespace, podName, err)
 	}
 
-	return status.Results{}, nil
+	const trafficSourcePort = 0
+	log.Printf("Running traffic for %dm...", int(e.testDuration.Minutes()))
+	_, err = trexClient.StartTraffic(ctx, e.trafficGeneratorPacketsPerSecondInMillions, trafficSourcePort, e.testDuration)
+	if err != nil {
+		return status.Results{}, fmt.Errorf("failed to run traffic from trex-console on pod \"%s/%s\" side: %w",
+			e.namespace, podName, err)
+	}
+	time.Sleep(e.testDuration)
```

Comment:
Please consider waiting on a channel here, while the test is running in a different goroutine.

Reply:
I generally agree but I am planning to remove this in the next PR anyway so I prefer not to bother, is that ok with you?

Reply:
Yes.